src/Containers/BikeMap/BikeMap.js

```diff
@@ -52,6 +54,15 @@ export class BikeMap extends Component {
             center={[30, 0]}
             zoom='3'>
             {this.markersToShow()}
+            {/* MapSearch is my withLeaflet wrapper on ReactLeafletZoomIndicator - I also tried wrapping it directly in this file */}
+            <MapSearch />
+            {/* ReactLeaflet search is the search bar component - see BikeMap.scss for styling I added */}
+            <ReactLeafletSearch 
```

Comment:
ReactLeaflet search is the search bar component - see BikeMap.scss for styling I added